distribution/ddf-common/src/main/resources/bin/ddf

```diff
@@ -14,20 +14,16 @@ HOME_DIR=$(cd "${SCRIPTDIR}/.."; pwd -P)
 # Useful files
 PROPERTIES_FILE=$HOME_DIR/etc/custom.system.properties
 RESTART_FILE="$SCRIPTDIR/restart.jvm"
-SOLR_EXEC=${SCRIPTDIR}/ddfsolr
 GET=${SCRIPTDIR}/get_property
 
 # Flags
 START_KARAF="false"
```

Comment:
Under what circumstances do we NOT start Karaf?

Reply:
You wouldn't.  I left it in case we would add another process back in but we would most likely do that with docker going forward so I can just remove this to clean it up.